applications/areg-portal/backend/main.py

```diff
@@ -23,11 +26,9 @@
 from api.controllers import *
 
 app = FastAPI(
-    description="areg_portal",
-    version="0.1.0",
-    title="areg_portal",
-    contact={'email': 'cloudharness@metacell.us'},
-    license={'name': 'UNLICENSED'},
+    title="Copy of areg-portal",
```

Comment:
Can use a better title -- Antibody Registry API as the description is OK

Reply:
Changed title to: 'SciCrunch Antibody Registry API'